gensim/corpora/sharded_corpus.py

```diff
@@ -789,8 +789,8 @@ def save(self, *args, **kwargs):
         if 'ignore' not in kwargs:
             kwargs['ignore'] = frozenset(attrs_to_ignore)
         else:
-            kwargs['ignore'] = frozenset([v for v in kwargs['ignore']]
-                                         + attrs_to_ignore)
+            kwargs['ignore'] = frozenset([v for v in kwargs['ignore']] +
+                                         attrs_to_ignore)
```

Comment:
Here `autopep8` changed the position of binary operator `+`. It is to correct the warning `W503`, i.e. *line break occurred before a binary operator*.